lib/components/atoms/Pagination/Pagination.style.js

```diff
@@ -0,0 +1,13 @@
+import { css } from 'styled-components';
+
+export default css`
+  li {
+    display: inline-block;
+    margin: 5px 5px 0 0;
+    margin: ${props => props.theme.spacing.unit2};
```

Comment:
You are working on old code the format for spacing was changed a week back 

unit2 --> UNIT_2

also you are setting 2 margin props in subsequent lines, think this should have been a padding